tests/transport_test.go

```diff
@@ -28,8 +29,11 @@ var _ = Describe("Transport Tests", func() {
 	)
 
 	var (
-		ns  string
-		f   = framework.NewFramework("transport", framework.Config{SkipNamespaceCreation: false})
+		ns string
+		f  = framework.NewFramework("transport", framework.Config{
```

Comment:
Isn't that the default config? skip namespace creation false, and honor wait for first consumer set?. I think we can just remove the config parameter.

Reply:
`Config{}` will set no feature gates

Reply:
Ah right, that is the same as default, just as before.  Assuming we wanted to be explicit here for a reason?

Reply:
No I think its a leftover from a long time ago. Feel free to remove it.